core/schema.sql

```diff
@@ -7,6 +7,7 @@
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
this is a PostgreSQL 9.6 config. we might want to remove it if other people are still using 9.5. or we could just require everyone to use 9.6 now?